api/src/index.js

```diff
@@ -29,3 +29,60 @@ app.use("/api", apiRouter);
 app.listen(process.env.PORT, () => {
   console.log(`API listening on port ${process.env.PORT}`);
 });
+//Future-meal
+app.get("/future-meals", async (req, res) => {
+  try {
+    const now = new Date();
+    const meals = await knex("Meal").where("when", ">", now);
```

Comment:
A suggestion would be to use knex `now` helper method to avoid being explicit about the date format

https://knexjs.org/guide/utility.html#now

```suggestion
    const meals = await knex("Meal").where("when", ">", knex.fn.now());
```